# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../shadow/shadow-4.1.5-nscd-newline-msg.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 360351b9b99bba0a278923c4237d158cbd726df5 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sun, 12 Feb 2012 23:08:42 -0500
Subject: [PATCH 1/3] nscd: add missing newline to warning msg

Trying to run `passwd` without nscd running shows the messages:
...
Re-enter new password:
passwd: nscd exited with status 1passwd: Failed to flush the nscd cache.
passwd: nscd exited with status 1passwd: Failed to flush the nscd cache.
passwd: nscd exited with status 1passwd: Failed to flush the nscd cache.
passwd: nscd exited with status 1passwd: Failed to flush the nscd cache.
passwd: password changed.
...

You can see the status message missing a newline, so add it.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 lib/nscd.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lib/nscd.c b/lib/nscd.c
index 227c205..ea1dc55 100644
--- a/lib/nscd.c
+++ b/lib/nscd.c
@@ -40,7 +40,7 @@ int nscd_flush_cache (const char *service)
 		   interpreter that is missing.  Probably the former. */
 		return 0;
 	} else if (code != 0) {
-		(void) fprintf (stderr, _("%s: nscd exited with status %d"),
+		(void) fprintf (stderr, _("%s: nscd exited with status %d\n"),
 		                Prog, code);
 		(void) fprintf (stderr, _(MSG_NSCD_FLUSH_CACHE_FAILED), Prog);
 		return -1;
-- 
1.7.8.4

